include/general.h

```diff
@@ -58,6 +66,9 @@ extern BYTE _oserror;
 
 #define VIDEOMODE (((*(BYTE *)0xD7) == 0x80) ? 80 : 40)
 
+#define EUR_FORMAT "%3d,%02d" EURSYM
+#define EUR_FORMAT_MINLEN (sizeof(EUR_FORMAT) + 1)
```

Comment:
`sizeof(EUR_FORMAT)` does not actually match the output — you’re wasting a byte?

I think it’d be cleaner to use e.g. `(sizeof("999,99" EURSYM) + 1)`

Reply:
Thanks for spotting this! :+1: